### PR TITLE
feat: added snackbar

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- added a snackbar component

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
small PR. Everything looks good. Can merge

File: components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import url(/stylesheets/main.css);
+
+.snackbar-container{
+    background-color: white;
+    border-radius: 4px;
+    width: 80%;
+
```
Comment (rittik-ghosh) [inline components/snackbar.css:7]: just an extra space

**Review by Sameony — APPROVED** [review]
short and concise code with the use of utility classes. good work

**Review by rittik-ghosh — APPROVED** [review]
All the code is good, clean and readable!!
Just an extra space is there but can be removed later.

**Pull request merged**